che-theia-plugins.yaml

```diff
@@ -768,3 +768,7 @@ plugins:
     repository:
       url: 'https://github.com/BroadcomMFD/cobol-control-flow'
     extension: https://open-vsx.org/api/BroadcomMFD/ccf/0.4.0/file/BroadcomMFD.ccf-0.4.0.vsix
+  - repository:
+      url: 'https://github.com/wholroyd/vscode-hcl'
+      revision: '0.0.5'
```

Comment:
I don't see `0.0.5` tag in https://github.com/wholroyd/vscode-hcl. Please see what `revision` attribute should describe: https://github.com/eclipse-che/che-plugin-registry/blob/main/CONTRIBUTE.md

Reply:
Ok, because of there are no tag on repo, I change revision to use last commit sha1 from master branch, aka `abe9c4122bd662ec30b8b5a9c73a00f6dc51e77d`. I hope is the right way.